includes/class-sensei-templates.php

```diff
@@ -567,19 +567,6 @@ public  static function deprecate_lesson_image_hook(){
     }// end sensei_deprecate_lesson_single_main_content_hook
 
     /**
-     * hook in the deprecated sensei_login_form hook for backwards
-     * compatibility
-     *
-     * @since 1.9.0
-     * @deprecated since 1.9.0
-     */
-    public static function deprecate_sensei_login_form_hook(){
```

Comment:
Adding a deprecated notice will be a way to notify plugin developers about the change and offer them an alternative.